packages/hash/api/src/model/knowledge/user.model.ts

```diff
@@ -30,57 +29,6 @@ type UserModelCreateParams = Omit<
  * @class {@link UserModel}
  */
 export default class extends EntityModel {
-  /**
-   * @todo: This method and `getUserByEntityId` is confusing. Should be fixed as part of:
-   *  https://app.asana.com/0/1200211978612931/1202937382769276/f
-   */
-  static async getUserByAccountId(
-    graphApi: GraphApi,
-    params: { accountId: string },
-  ): Promise<UserModel | null> {
-    const allUsers = await EntityModel.getByQuery(graphApi, {
-      all: [
-        { eq: [{ path: ["version"] }, { literal: "latest" }] },
-        {
-          eq: [
-            { path: ["type", "versionedUri"] },
-            { literal: WORKSPACE_TYPES.entityType.user.schema.$id },
-          ],
-        },
-      ],
-    });
-
-    const matchingUser = allUsers
-      .filter(({ accountId }) => accountId === workspaceAccountId)
-      .map(UserModel.fromEntityModel)
-      .find((user) => user.entityId === params.accountId);
-
-    return matchingUser ?? null;
-  }
-
-  /**
-   * Get a workspace user entity by their entityId.
-   *
-   * @param params.entityId - the entityId of the user
-   */
-  static async getUserByEntityId(
```

Comment:
This method's implementation is identical to `getUserById` (except for its return-type, which was incorrect anyway), so have removed it.